STEER/ESD/AliESDInputHandlerRP.cxx

```diff
@@ -99,8 +99,9 @@ Bool_t AliESDInputHandlerRP::Init(Option_t* opt)
     {
       TString rppath;
       if (det->TestBit(kReadFromArchiveBIT)) {
-	// if (fPathName->EndsWith("root_archive.zip"))
-	if (fPathName->Contains("root_archive"))
+	if (fPathName->Contains("root_archive", TString::kIgnoreCase) &&
+	    fPathName->Contains(".zip", TString::kIgnoreCase))
```

Comment:
No, EndsWith would exclude other cases, such as 
```
root_archive.zip.001
```
I think the library is agnostic to the _ending_ so it shouldn't really matter. 

Of course, by far the best option would be _not_ to rely on the file name but instead use file magic - but alas ROOT does not provide that service. 

Christian